tests/integration/backends/test_tpp.py

```diff
@@ -0,0 +1,379 @@
+from datetime import date
+
+import pytest
+import sqlalchemy
+
+from databuilder.backends.tpp import TPPBackend
+from databuilder.tables.beta import tpp
+from tests.lib.tpp_schema import (
+    APCS,
+    EC,
+    APCS_Der,
+    CodedEvent,
+    CodedEventSnomed,
+    EC_Diagnosis,
+    HealthCareWorker,
+    MedicationDictionary,
+    MedicationIssue,
+    ONSDeaths,
+    Organisation,
+    Patient,
+    PatientAddress,
+    PotentialCareHomeAddress,
+    RegistrationHistory,
+    SGSS_AllTests_Negative,
+    SGSS_AllTests_Positive,
+    Vaccination,
+    VaccinationReference,
+)
```

Comment:
Oh yes, this thought occurred to me too but then I forgot about it. I'm sure there's some way of doing this and I think it would be valuable. I'll revisit before merging.